Docs/user_guide/get_help.md

```diff
@@ -0,0 +1,31 @@
+# Get Help
+
+[Contact the Slicer community or commercial partners](about.html#contact-us) if you have any questions, bug reports, or enhancement requests.
+
+## I need help in using Slicer
+
+- I have a question about how to use Slicer for a particular task or I'm not sure things are working right:
+  - Type your queston into Google. There is a good chance that your question has been asked and anwered before and all questions ever asked about Slicer are publicly available and indexed by Google.
+  - Try your best to sort out the issue by reading documentation and checking error logs
+    - User and developer manual are available at https://slicer.readthedocs.io
+    - Look at our [portfolio of training materials](https://www.slicer.org/wiki/Documentation/Nightly/Training) and make a google search to see if the question has been discussed.
```

Comment:
Again same as an above comment about whether to search google vs the Slicer discourse forum directly